### DISCUSSION TITLE
ScrollArea scroll snap

### DISCUSSION THREAD

**Discussion by khalibloo** [mantinedev/mantine#2665] [Feature requests]
ScrollArea doesn't support scroll snapping even through the styles API, as the immediate parent (div) of the content isn't targetable. Ideally, this should be supported through simple props, but for added flexibility, the immediate parent of the content should also be exposed in the styles API.

**Comment by rtivital** [comment]
As far as I know, ScrollArea component scrolling is managed by JavaScript, scroll snaps will not work.

**Reply by khalibloo** [reply]
I see... If there was a way to reuse the look of the scrollbars, it would be fairly easy to just use a Box with some styling to get the snap behavior. But it appears the scroll bars are actual DOM elements too, not CSS

**Reply by rtivital** [reply]
Well, that's the entire idea of ScrollArea – it makes scrollbars look the same in all browsers
